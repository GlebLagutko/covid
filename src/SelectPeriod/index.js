import React from "react";

function SelectPeriod({period, setPeriod}) {

    return (
        <select value={period} onChange={e => setPeriod(e.target.value)}>
            <option value='Year' key='Year'>Year</option>)
            <option value='OneDay' key='OneDay'>OneDay</option>)
        </select>
    );
}

export default SelectPeriod;