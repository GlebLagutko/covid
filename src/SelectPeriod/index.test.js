import {render} from '@testing-library/react';

import SelectPeriod from '.';
import React from "react";

describe('SelectPeriod', () => {

  it('renders correctly', async () => {

    const {container} = render(<SelectPeriod/>);

    expect(container).toMatchSnapshot();
  });
});