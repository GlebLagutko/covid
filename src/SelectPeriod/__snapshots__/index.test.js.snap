// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectPeriod renders correctly 1`] = `
<div>
  <select>
    <option
      value="Year"
    >
      Year
    </option>
    )
    <option
      value="OneDay"
    >
      OneDay
    </option>
    )
  </select>
</div>
`;
